src/main/java/datastax/astra/migrate/BaseJobSession.java

```diff
@@ -55,6 +59,11 @@ public abstract class BaseJobSession {
     protected Integer filterColIndex;
     protected String filterColValue;
 
+    protected BaseJobSession(SparkConf sc) {
+        readConsistencyLevel = Util.mapToConsistencyLevel(Util.getSparkPropOrEmpty(sc, "spark.consistency.read"));
+        writeConsistencyLevel = Util.mapToConsistencyLevel(Util.getSparkPropOrEmpty(sc, "spark.consistency.write"));
```

Comment:
Question here @pravinbhat -- With Astra DB, only `LOCAL_QUORUM` / `LOCAL_SERIAL` is supported for writes. Beginning [yesterday](https://www.datastax.com/blog/enhanced-multi-region-database-consistency-astra-db), we also support `EACH_QUORUM` and `SERIAL` on the write consistency. All other consistency levels will fail.

Either we could do that check or leave it as-is and let it fail and the user could update with the correct write CL. WDYT?

Reply:
@msmygit The tool is not `Astra` specific, we have cleaned up some Astra specific code & plan to do additional refactor to remove any signs of Astra in variable naming (i.e. we plan to use `origin` & `target` everywhere).

In most use-cases (& specially for Astra use-case) it would be the default, which is now `LOCAL_QUORUM` (after this PR). This PR was mainly to fix inconsistency, i.e. some queries used `LOCAL_QUORUM` & some did not.

Reply:
https://docs.datastax.com/en/dse/5.1/dse-dev/datastax_enterprise/spark/sparkCassandraProperties.html - Isn't the default CL for writes is LOCAL_QUORUM? Why would some queries use LOCAL_QUORUM and some wouldn't?